RobotStateMessage currentState
RobotStateMessage targetState
---
float64 distance